### PR TITLE
Explicit cipher close

### PR DESCRIPTION
**Pull request by sundapeng**
Since some re-factory have been done at https://github.com/intel-hadoop/chimera/pull/20, the pull request is based on the commits of  https://github.com/intel-hadoop/chimera/pull/20.
The commit `Explicit close() on Cipher  e5f74cf` is real pull request

### CODE REVIEW

File: src/main/java/com/intel/chimera/CryptoInputStream.java [hunk 1]
```diff
@@ -43,6 +45,8 @@
  */
 public class CryptoInputStream extends InputStream implements
     ReadableByteChannel {
+  public static final Log LOG = LogFactory.getLog(CryptoInputStream.class);
```
Comment (jerrychenhf) [inline src/main/java/com/intel/chimera/CryptoInputStream.java]: @sundapeng The change is overall good.
One minor question: LOG is not used anywhere, is there any reason adding this? I saw CryptoOutputStream doesn't have a LOG.

Comment (sundapeng) [inline src/main/java/com/intel/chimera/CryptoInputStream.java]: Thank @jerrychenhf for catching it. I will remove it.

**Comment by jerrychenhf** [comment]
+1

**Comment by sundapeng** [comment]
Thank @jerrychenhf for your review, merged to master

**Pull request merged**